docs/index.rst

```diff
@@ -3,298 +3,57 @@
 
-The more general case where the detector is not aligned with north, would have
-a rotation transform after the pixelshift and pixelscale transformations to
-align the detector coordinates with north up.
 
-The net transformation from pixel coordinates to celestial coordinates then
-becomes:
+Introductions
+-------------
 
-.. doctest-skip::
+For Users
+*********
 
-  >>> det2sky = pixelshift | pixelscale | tangent_projection | celestial_rotation
-
-The remaining elements to defining the WCS are he input and output
-frames of reference. While the GWCS scheme allows intermediate frames
-of reference, this example doesn't have any. The output frame is
-expressed with no associated transform
-
-.. doctest-skip::
-
-  >>> detector_frame = cf.Frame2D(name="detector", axes_names=("x", "y"),
-  ...                             unit=(u.pix, u.pix))
-  >>> sky_frame = cf.CelestialFrame(reference_frame=coord.ICRS(), name='icrs',
-  ...                               unit=(u.deg, u.deg))
-  >>> wcsobj = wcs.WCS([(detector_frame, det2sky),
-  ...                   (sky_frame, None)
-  ...                  ])
-  >>> print(wcsobj)
-    From      Transform
-  -------- ----------------
-  detector detector_to_sky
-      icrs             None
-
-To convert a pixel (x, y) = (1, 2) to sky coordinates, call the WCS object as a function:
-
-.. doctest-skip::
-
-  >>> sky = wcsobj(1, 2)
-  >>> print(sky)
-  (29.980402161089177, 44.98616499109102)
-
-The :meth:`~gwcs.wcs.WCS.invert` method evaluates the :meth:`~gwcs.wcs.WCS.backward_transform`
-if available, otherwise applies an iterative method to calculate the reverse coordinates.
-
-GWCS supports the :ref:`wcsapi` which defines several methods to work with high level Astropy objects:
-
-.. doctest-skip::
-
-  >>> sky_obj = wcsobj.pixel_to_world(1, 2)
-  >>> print(sky)
-  <SkyCoord (ICRS): (ra, dec) in deg
-    (29.98040216, 44.98616499)>
-  >>> wcsobj.world_to_pixel(sky_obj)
-  (0.9999999996185807, 1.999999999186798)
-
-.. _save_as_asdf:
-
-Save a WCS object as a pure ASDF file
-+++++++++++++++++++++++++++++++++++++
-
-.. doctest-skip::
-
-  >>> from asdf import AsdfFile
-  >>> tree = {"wcs": wcsobj}
-  >>> wcs_file = AsdfFile(tree)
-  >>> wcs_file.write_to("imaging_wcs.asdf")
-
-
-:ref:`pure_asdf`
-
-
-Reading a WCS object from a file
-++++++++++++++++++++++++++++++++
-
-
-`ASDF <https://asdf.readthedocs.io/>`__ is used to read a WCS object
-from a pure ASDF file or from an ASDF extension in a FITS file.
-
-
-.. doctest-skip::
+.. toctree::
+  :maxdepth: 3
 
-  >>> import asdf
-  >>> asdf_file = asdf.open("imaging_wcs.asdf")
-  >>> wcsobj = asdf_file.tree['wcs']
+  gwcs/user_introduction.rst
 
 
-Other Examples
---------------
+For Developers
+**************
 
 .. toctree::
-  :maxdepth: 2
+  :maxdepth: 3
 
+  gwcs/developer_introduction.rst
   gwcs/imaging_with_distortion.rst
   gwcs/ifu.rst
+  gwcs/api.rst
 
 
-
-Using ``gwcs``
---------------
+Using GWCS
+----------
 
 .. toctree::
   :maxdepth: 2
```

Comment:
Wouldn't the `using_wcs.rst` and maybe `wcstools.rst` be better under the new user guide section?